apis/container/v1alpha1/nodepool_types.go

```diff
@@ -568,7 +568,7 @@ type NodePoolStatus struct {
 // +kubebuilder:printcolumn:name="READY",type="string",JSONPath=".status.conditions[?(@.type=='Ready')].status"
 // +kubebuilder:printcolumn:name="SYNCED",type="string",JSONPath=".status.conditions[?(@.type=='Synced')].status"
 // +kubebuilder:printcolumn:name="STATE",type="string",JSONPath=".status.atProvider.status"
-// +kubebuilder:printcolumn:name="CLUSTER-NAME",type="string",JSONPath=".spec.forProvider.cluster"
+// +kubebuilder:printcolumn:name="CLUSTER-NAME",type="string",JSONPath=".spec.forProvider.clusterRef.name"
```

Comment:
```suggestion
// +kubebuilder:printcolumn:name="CLUSTER-REF",type="string",JSONPath=".spec.forProvider.clusterRef.name"
```
What would you think about changing like this? Might be a little less confusing if using a cluster that was not provisioned by Crossplane.

Reply:
Yeah, that sounds more accurate. Thanks!